[2023-11-12T08:02:16.789+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v2.insert_into_table scheduled__2023-11-05T00:00:00+00:00 [queued]>
[2023-11-12T08:02:16.807+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v2.insert_into_table scheduled__2023-11-05T00:00:00+00:00 [queued]>
[2023-11-12T08:02:16.808+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-11-12T08:02:16.839+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2023-11-05 00:00:00+00:00
[2023-11-12T08:02:16.849+0000] {standard_task_runner.py:57} INFO - Started process 9290 to run task
[2023-11-12T08:02:16.864+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v2', 'insert_into_table', 'scheduled__2023-11-05T00:00:00+00:00', '--job-id', '95', '--raw', '--subdir', 'DAGS_FOLDER/dag-posgres-operator.py', '--cfg-path', '/tmp/tmpsxwnb_kx']
[2023-11-12T08:02:16.869+0000] {standard_task_runner.py:85} INFO - Job 95: Subtask insert_into_table
[2023-11-12T08:02:16.973+0000] {task_command.py:416} INFO - Running <TaskInstance: dag_with_postgres_operator_v2.insert_into_table scheduled__2023-11-05T00:00:00+00:00 [running]> on host aac4306c1949
[2023-11-12T08:02:17.170+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='nguyenthung' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v2' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2023-11-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-05T00:00:00+00:00'
[2023-11-12T08:02:17.171+0000] {sql.py:274} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2023-11-05', 'dag_with_postgres_operator_v2')
        
[2023-11-12T08:02:17.185+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-11-12T08:02:17.199+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-11-12T08:02:17.207+0000] {sql.py:418} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2023-11-05', 'dag_with_postgres_operator_v2')
        , parameters: None
[2023-11-12T08:02:17.208+0000] {sql.py:427} INFO - Rows affected: 1
[2023-11-12T08:02:17.236+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v2, task_id=insert_into_table, execution_date=20231105T000000, start_date=20231112T080216, end_date=20231112T080217
[2023-11-12T08:02:17.275+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-12T08:02:17.307+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
